docs/guides/Integration-Guides/erp-integration-guide/import-customer-data.md

```diff
@@ -8,35 +8,20 @@ updatedAt: '2022-06-01T21:27:30.468Z'
 
 When implementing an ecommerce project, it may be necessary to import customer data from other systems, such as Customer Relationship Management (CRM) applications.
 
-You can use [Master Data v2](https://help.vtex.com/en/tutorial/master-data-v2--3JJ1mlzuo88w22gO0gy0QS#) to manage many types of data and it provides the best way of integrating your customer data base to VTEX.
+VTEX Checkout natively stores shopper data in [Master Data v1](https://help.vtex.com/en/tutorial/master-data--4otjBnR27u4WUIciQsmkAw). Because of this, using the [Master Data v1 API](https://developers.vtex.com/docs/api-reference/masterdata-api) is the best way of integrating your customer data base to VTEX.
```

Comment:
```suggestion
The **Checkout** module stores shopper data in [**Master Data v1**](https://help.vtex.com/en/tutorial/master-data--4otjBnR27u4WUIciQsmkAw) by default. Because of this, using the [**Master Data v1** API](https://developers.vtex.com/docs/api-reference/masterdata-api) is the best way to integrate your customer database into VTEX.
```